README.md

```diff
@@ -94,27 +94,21 @@ at any position since RedisInsight will sort them anyway.
     ]
 }
 ```
-However, it is better to use any comfortable order or "rules" to make manage process
-more comfortable.
 
-After marge into `latest` branch and RedisInsight sync with `notifications.json`
-you will see it as a toast message in the left corner of RedisInsight app.
+After making all the changes, merge the updated `notifications.json` into the `latest` branch. RedisInsight checks new notifications on the application start-up or each 1 hour (at '00) if the application is not closed. When changes are captured, RedisInsight will display the notification.
```

Comment:
The flow is to merge changes from feature* branch to main. Only after all changes made and merged to main branch we must merge it to the latest branch. So we have a such flow features* -> main -> (double check everything) -> latest (auto release and ready for autoupdates)